Source/Core/Logging/FramesLogEvent.swift

```diff
@@ -49,71 +48,68 @@ enum FramesLogEvent: Equatable, PropertyProviding {
-             .threeDSChallengeComplete(let success, _):
-            return success ? .info : .error
+            case .paymentFormInitialised, .paymentFormPresented, .billingFormPresented, .threeDSWebviewPresented:
+                return .info
+            case .exception:
+                return .error
+            case .threeDSChallengeLoaded(let success), .threeDSChallengeComplete(let success, _):
+                return success ? .info : .error
         }
     }
 
     var properties: [Property: AnyCodable] {
         switch self {
-        case .billingFormPresented,
-             .threeDSWebviewPresented:
-            return [:]
-        case .paymentFormPresented:
-            return [Property.locale: Locale.current.identifier].mapValues(AnyCodable.init(_:))
-        case let .paymentFormInitialised(environment):
-            let environmentString = environment.rawValue == "live" ? "production" : environment.rawValue
-            return [.environment: environmentString].mapValues(AnyCodable.init(_:))
-        case let .threeDSChallengeLoaded(success):
-            return [.success: success].mapValues(AnyCodable.init(_:))
-        case let .threeDSChallengeComplete(success, tokenID):
-            return [.success: success]
-                .updating(key: .tokenID, value: tokenID)
-                .mapValues(AnyCodable.init(_:))
-        case let .exception(message):
-            return [.message: message]
-                .mapValues(AnyCodable.init(_:))
+            case .billingFormPresented, .threeDSWebviewPresented:
+                return [:]
+            case .paymentFormPresented:
+                return [Property.locale: Locale.current.identifier].mapValues(AnyCodable.init(_:))
+            case let .paymentFormInitialised(environment):
+                let environmentString = environment.rawValue == "live" ? "production" : environment.rawValue
+                return [.environment: environmentString].mapValues(AnyCodable.init(_:))
+            case let .threeDSChallengeLoaded(success):
+                return [.success: success].mapValues(AnyCodable.init(_:))
+            case let .threeDSChallengeComplete(success, tokenID):
+                return [.success: success]
+                    .updating(key: .tokenID, value: tokenID)
+                    .mapValues(AnyCodable.init(_:))
+            case let .exception(message):
+                return [.message: message]
+                    .mapValues(AnyCodable.init(_:))
         }
     }
 
     var loggedOncePerCorrelationID: Bool {
         switch self {
-        case .paymentFormInitialised,
-             .paymentFormPresented,
-             .billingFormPresented,
-             .threeDSWebviewPresented:
-            return true
-        case .threeDSChallengeLoaded,
-             .threeDSChallengeComplete,
-             .exception:
-            return false
+            case
+                .paymentFormInitialised,
+                    .paymentFormPresented,
```

Comment:
This is getting silly with the parameter realignment, should we undo whatever made this happen?

Reply:
Currently @ehab-al-cko has put in place what is suggested from RW guide as default. But I can see there a few things we may not really like from RW guide and I have suggested @ehab-al-cko to discuss this in the next dev meeting or earlier. if you like to and decide all at once what rules you are comfortable with. For me all the formatting changes are just a matter of agreement between devs ( Happy with whatever you both decide 👍 )